README.md

```diff
@@ -122,22 +122,21 @@ A very fast version of Isochrone API. However, the request parameters are much m
 
 ```typescript
 import {
-  TimeMapFastRequestArrivalSearch,
+  TimeMapFastRequestSearch, TravelTimeClient,
```

Comment:
lets get rid of Client import, since at this point of code we already assume that client is initialised